src/sql/workbench/contrib/notebook/browser/models/outputProcessor.ts

```diff
@@ -78,7 +78,7 @@ export function getBundleOptions(options: IOutputModelOptions): MimeModel.IOptio
  */
 export function extract(value: JSONObject, key: string): {} {
 	let item = value[key];
-	if (isPrimitive(item)) {
```

Comment:
bug fix. value of undefined and null note being handled correctly. It is not possible to do JSON.parese on undefined.